packages/flutter/lib/src/foundation/diagnostics.dart

```diff
@@ -1470,7 +1471,7 @@ abstract class DiagnosticsNode {
   /// the value returned here but other factors also influence it. For example,
   /// whether an exception is thrown computing a property value
   /// [DiagnosticLevel.error] is returned.
-  DiagnosticLevel get level => DiagnosticLevel.info;
+  DiagnosticLevel get level => kReleaseMode ? DiagnosticLevel.hidden : DiagnosticLevel.info;
```

Comment:
This also seems odd for the same reasons...

Reply:
This definitely makes a difference today.

Reply:
(For AOT mode and Dart2js apparently)

Reply:
There are other getters for level. We should probably also override them for release mode.